source/dbo/Stored Procedures/Power BI Reporting/uspMapCaseRecordChestXrayResults.sql

```diff
@@ -0,0 +1,36 @@
+﻿CREATE PROCEDURE [dbo].[uspMapCaseRecordChestXrayResults]
+AS
+BEGIN TRY
+	WITH rankedResults AS
+	(
+	SELECT mr.NotificationId
+	,mr.Result AS result
+	,ROW_NUMBER() OVER
+		(PARTITION BY mr.NotificationId
+		ORDER BY ABS(DATEDIFF(day, mr.TestDate, cd.DiagnosisDate)) ASC
+		,CASE WHEN Result = 'ConsistentWithTbCavities' THEN 1
+			WHEN Result = 'ConsistentWithTbOther' THEN 2
+			WHEN Result = 'NotConsistentWithTb' THEN 3
+			WHEN Result = 'Awaiting' THEN 4 END ASC
+		) AS rn
+	FROM [$(NTBS)].[dbo].[ManualTestResult] mr
+		INNER JOIN [dbo].[Record_CaseData] cd ON cd.NotificationId = mr.NotificationId
```

Comment:
I would add an extra clause here and SourceSystem = 'NTBS' - just belt and braces as Record_CaseData contains ETS records as well. The inner join should never connect ETS data to an NTBS record but it is just a failsafe.

Reply:
Record_CaseData doesn't have a source field, it's RecordRegister that does. It would make sense to have this check, maybe we should add in the source into Record_CaseData?